lib/bootstrap.js

```diff
@@ -27,6 +27,10 @@ var debug = require('debuglog')('kraken/bootstrap');
 
 module.exports = function (app, options) {
 
+    // on bootstrapinit
+    const onbootstrapinit = options.onbootstrapinit;
```

Comment:
My understanding is that this is invoked when kraken is mounted in an express app and express app instance is available to pass down the callback function. Thoughts on calling this as `onkrakenmount` or `onmount`?